fvtest/porttest/omrstrTest.cpp

```diff
@@ -179,46 +179,33 @@ test_omrstr_vprintf(struct OMRPortLibrary *portLibrary, const char *testName, co
 	/* Buffer larger than required */
 	va_start(args, expectedResult);
```

Comment:
va_list variables are effectively const in methods that instantiate them -- these guys just need to call va_start() once and balance it with va_end() after use. So it can be used multiple times in declaring method and in called methods that do not call va_arg() to iterate the contained vars. 

This is so as long as called methods use va_arg() to obtain vars call va_copy() and iterate the copy, leaving the caller's original instance untouched.